Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -2052,6 +2057,8 @@
     <EmbeddedResource Include="$(MSBuildThisFileDirectory)Issue12084.xaml">
       <Generator>MSBuild:UpdateDesignTimeXaml</Generator>
     </EmbeddedResource>
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)Issue12473.xaml">
+      <SubType>Designer</SubType>
```

Comment:
you need to close the `EmbeddedResource` tag here

Reply:
Fixed! Thank you for catching that!!